data/basics/userBalance.go

```diff
@@ -96,15 +96,29 @@ func UnmarshalStatus(value string) (s Status, err error) {
 	return
 }
 
+// AgreementAccountData contains just the account data used by agreement.
+type AgreementAccountData struct {
+	_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+	Status     Status     `codec:"onl"`
```

Comment:
move `Status` to `AccountData`

Reply:
Done. It was used in `VotingStake` to return zero if not Online. I moved this logic into `OnlineAccountData()` helper method. Agreement in theory might receive a malformed vote from the network, so this safety check looks important.